src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -154,7 +206,7 @@
      * includes validation
      */
     public static Object externalValueToInternalValue(GraphqlFieldVisibility fieldVisibility, Object externalValue, GraphQLInputType type) {
-        return new ValuesResolver().externalValueToInternalValue(fieldVisibility, type, externalValue, ValueMode.COERCED);
+        return new ValuesResolver().externalValueToInternalValue(fieldVisibility, type, externalValue);
```

Comment:
I think this is a bad pattern BTW - that is ValuesResolver is a stateless class - so make it method static.

No object allocation needed - even with escape analysis its a bit better

Reply:
Yeah .. not really happy about it myself... changed it